metadata-ingestion/tests/integration/feast/serving/online-serving.yml

```diff
@@ -0,0 +1,6 @@
+COMPOSE_PROJECT_NAME=feast
+FEAST_VERSION=develop
+FEAST_CORE_CONFIG=./core/core.yml
+FEAST_ONLINE_SERVING_CONFIG=./serving/online-serving.yml
+GCP_SERVICE_ACCOUNT=./gcp-service-accounts/placeholder.json
+INGESTION_JAR_PATH=https://storage.googleapis.com/feast-jobs/spark/ingestion/feast-ingestion-spark-develop.jar
```

Comment:
interesting tht they host this on google

Reply:
Yeah... not sure why 🤷